pipelines/timeseries_forecasts.R

```diff
@@ -250,10 +253,12 @@ disease_name_nssp_map <- c(
   "influenza" = "Influenza"
 )
 
-base_dir <- path_dir(model_run_dir)
-
-disease_name_raw <- base_dir |>
-  path_file() |>
+# replace this with functionality from hewr
+disease_name_raw <- model_run_dir |>
+  path_split() |>
+  pluck(1) |>
+  tail(3) |>
+  head(1) |>
```

Comment:
As above, use `hewr` now.

Reply:
Resolved in https://github.com/CDCgov/pyrenew-hew/pull/136/commits/3f8139b920dbba75c425e6851be835b47d22f050